frontend/public/src/components/EnrolledItemCard_test.js

```diff
@@ -251,25 +245,6 @@ describe("EnrolledItemCard", () => {
       assert.isTrue(verifiedUnenrollmodal.exists())
     })
   })
-  ;[[true], [false]].forEach(([approvedFlexiblePrice]) => {
-    it("renders the financial assistance link", async () => {
```

Comment:
Why was this test removed?

Reply:
I removed the FA assistance link from the enrolled card, as per design.